### PR TITLE
UGC-4760 | Add load faster variant for higlight edit section button

### PR DESCRIPTION
**Pull request by MrVanosh**

### CODE REVIEW

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js [hunk 1]
```diff
@@ -1027,6 +1027,25 @@ var isExperimentTrackingSuccess = false;
...
+					const config = { childList: true, subtree: true };
+
+					const mutationObserver = new MutationObserver((mutationsList, observer) => {
+						mutationsList.forEach((mutation) => {
+							if (mutation.type === 'childList' && mutation.addedNodes.length > 0) {
+								const addedNode = mutation.addedNodes[0];
+								if (addedNode && addedNode.querySelector('[data-testid="highlight-action_edit-section"]')) {
+									this.preloadModules();
+									const editButton = addedNode.querySelector('[data-testid="highlight-action_edit-section"]');
+									editButton.addEventListener('click', init.onEditTabClick.bind(init, 'visual'));
```
Comment (kasperekt) [inline modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js]: suggestion: Could we disconnect the MutationObserver after we reach this block?

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js [hunk 2]
```diff
@@ -1027,6 +1027,25 @@ var isExperimentTrackingSuccess = false;
 				if ( isUserInExperiment() && !isUserInControlGroup() ) {
 					$('.mw-editsection').off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
 					$('.page-side-edit').off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
+
+					const wrapperNode = document.querySelector('.highlight__actions');
+					const config = { childList: true, subtree: true };
```
Comment (kasperekt) [inline modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js]: **question:** Do we need to have `subtree` here? Do we expect new nodes to appear deeper than on child level of .highlight__actions? Maybe we could observe another element instead?

**Review by kasperekt — CHANGES_REQUESTED** [review]

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js (hunk 1 above)
Comment (MrVanosh) [reply modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js]: Added

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js (hunk 2 above)
Comment (MrVanosh) [reply modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js]: It was deeper but I see this works without subtree. Changed

**Review by dariuszpaluch — APPROVED** [review]

**Pull request merged**